### DISCUSSION TITLE
Approval for WalletConnect explorer: UPBOND Wallet

### DISCUSSION THREAD

**Discussion by bunbuarka** [WalletConnect/walletconnect-monorepo#1632] [Developer Support]
Hi WalletConnect team!

could you please help to review our application for WalletConnect explorer registry
our Wallet name is UPBOND Wallet.
Thank you!

**Comment by bunbuarka** [comment]
Hi WalletConnect Team!

please kindly check the latest submission from UPBOND Wallet

**Comment by bunbuarka** [comment]
Hello,
anyone can help?

**Comment by bunbuarka** [comment]
Hi @bkrem, 

apologize for mentioning you out of nowhere, but I see that you responded another similar request. can you please kindly help our project submission?

uuid=75d95634-6e49-4310-a3ba-4e826dc5db5f

thank you very much

**Reply by bkrem** [reply]
Hi @bunbuarka,

Apologies about the delay here, we're currently taking active steps to avoid lengthy review times, so shouldn't happen again I hope 🙏 

I just checked that UUID and it seems to be a 404/not found. The only "Pending review" entry I see related to UPBOND is `5cb8b1a7-3901-44a5-b097-7c10d946e283`, which seems to have been submitted from a SIWE-based account so I can't verify authenticity via the creator email.

If this is the correct/wanted submission, could you please email me at ben@walletconnect.com from your `@upbond.io` domain so I can move ahead with the review?

P.S. for yourselves (and any other interested parties): The WalletConnect team is currently prioritising submissions that support WalletConnect Sign v2 or our newer, higher-level [Web3Wallet SDK](https://docs.walletconnect.com/2.0/web3wallet/about).

**Reply by bunbuarka** [reply]
Thank you for the response! 
email sent!
